CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+# Contributing to Bugwarrior
+
+Thank you for your interest in contributing to Bugwarrior!
+
+See the full [contributing documentation](https://bugwarrior.readthedocs.io/en/latest/contributing.html) or view it locally in `bugwarrior/docs/contributing`.
```

Comment:
Could you adjust the link to point to the current docs at https://bugwarrior-docs.readthedocs.io/en/latest/contributing/getting-started.html? See #812 for the unfortunate history behind this.

Reply:
done!